doc/api/next_api_changes/2018-09-15-AL.rst

```diff
@@ -0,0 +1,8 @@
+API changes
+```````````
```

Comment:
Please adapt to the new API change doc structure.

Reply:
fixed